client/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "npm run clean && webpack --config configuration/webpack/webpack.config.prod.js",
     "clean": "rimraf build",
     "dev": "npm run clean && webpack --config configuration/webpack/webpack.config.dev.js",
-    "e2e": "jest e2e",
+    "e2e": "jest --verbose false --config __tests__/e2e/e2e-jest-config.json e2e",
```

Comment:
Nit:  the change to the command line `unit-test` target (which happened on a prior PR) broke the ability to run single tests from the command line.   We used to be able to ay `npm test client/__tests__/foo.test.js` and it would run just that single test.   It looks like the way `jest` interprets the --testPathIgnorePatterns is incompatible with that.   If you see an easy fix, it would make it very convenient for unit test authors (the alternative is to use `node_modules/.bin/jest testname`.

Reply:
I bet I can fix this. Let me look into it.

Reply:
I added a script named "test" that just runs `jest` (this is the way it used to be) to fix your cli workflow.  We just won't use this script in the CI.

Reply:
sounds reasonable.  if I understand, this will run all tests if executed as `npm run test`, but with a specific test argument, will work as it used to.   Thanks!